crates/driver/src/boundary/settlement.rs

```diff
@@ -171,12 +169,18 @@ impl Settlement {
             }
             settlement::Internalization::Disable => InternalizationStrategy::EncodeAllInteractions,
         });
-        let builder = settle_method_builder(
-            contract,
-            encoded_settlement,
-            ethcontract::Account::Local(self.solver.into(), None),
-        );
-        let tx = builder.into_inner();
+
+        let account = ethcontract::Account::Local(self.solver.into(), None);
```

Comment:
Moved from `settlement_simulation` module since this was the only code that was still in use.